site/docs/master/restic.md

```diff
@@ -418,6 +418,12 @@ Velero does not currently provide a mechanism to detect persistent volume claims
 
 To solve this, a controller was written by Thomann Bits&Beats: [velero-pvc-watcher][7]
 
+### Add backup annotation
+
+Velero hasn't provided a do-one-command or automatic way to backup all volume resources in the cluster without annotation.
+
+To solve this, [velero-volume-controller][10] helps to achieve this goal by adding backup annotation to pods with volumes automatically.
```

Comment:
Some changes I would suggest in bold:

>The [velero-volume-controller][10] **helps to solve this problem** by adding **the** backup annotation to pods with volumes automatically.

I also agree with Carlisia's request for attribution.